### ISSUE TITLE
Issue with quoted text

### ISSUE THREAD

**Issue by crew102** [CSSIP-AIR/PatentsView-DB#4]
It looks like there is an issue with how PatentsView handles quotation marks. For example, whenever a quotation mark occurs in the patent's title, PatentsView quotes the entire title and adds extra quotation marks around the actual quoted text. You can see this behavior in patent number [5767337](https://www.google.sr/patents/US5767337):

```r
library(patentsview)

title <- search_pv(
    query = '{"_eq":{"patent_number":"5767337"}}'
)$data$patents$patent_title

cat(title)
#> "Creation of human apolipoprotein E isoform specific transgenic mice in apolipoprotein deficient ""knockout"" mice"
```

The same behavior is seen in the bulk data files.

**Comment by sarahkelley** [comment]
Yes, that indeed seems like a bug! Thanks for the heads up, we will let you know when it is corrected!

**Comment by Everst** [comment]
Thanks for responding Sarah. The reason for this bug is because of quote escapes as we parse first into tsv and only then import into MySQL. Fields separated by quotes. I suggest we fix that at the DB stage as part of our transformation routine.

**Comment by sarahkelley** [comment]
Thanks for the suggestion Evgeny, that makes a lot of sense!

**Comment by Everst** [comment]
Sarah: an even easier solution - use mediumtext or longtext in the DB to store detailed descriptions http://boolean.co.nz/blog/max-length-for-mysql-text-field-types/135/
